### PR TITLE
Update service-gateway-onprem-manage-oracle.md

### PR DESCRIPTION
**Pull request by alexkeh**
Updated on-prem gateway instructions for Oracle DB to reflect new method to install and configure Oracle clients for MS tools with the Oracle Client for MS Tools.

### CODE REVIEW

**Comment by alexkeh** [comment]
#sign-off

**Comment by American-Dipper** [comment]
@arthiriyer - can you review the proposed changes?  

 
IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.  

 

#label:"aq-pr-triaged" 

@MicrosoftDocs/public-repo-pr-review-team

**Comment by maggiesMSFT** [comment]
Thanks, @alexkeh 
#sign-off

**Comment by maggiesMSFT** [comment]
Hey, @alexkeh -- are you a MSFT employee, vendor, or contractor? If so you should be editing articles in the private repo. Start here:
https://aka.ms/savetime
That gets you into the private repo.
Enter the URL of the article you want to edit.

**Comment by alexkeh** [comment]
@maggiesMSFT I work at a partner. company, Oracle. I am not a MS employee, vendor, nor contractor.

Nonetheless, I tried to access the site with my MS credentials. It says I don't have permission to the resource.

File: powerbi-docs/connect-data/service-gateway-onprem-manage-oracle.md [hunk 1]
```diff
@@ -20,29 +20,23 @@ After you [install the on-premises data gateway](/data-integration/gateway/servi
...
 
 1. On the **New connection** screen for your on-premises data gateway, select **Oracle** for **Connection type**.
 
    :::image type="content" source="media/service-gateway-onprem-manage-oracle/add-data-source-oracle.png" alt-text="Screenshot of adding an Oracle data source to the gateway." :::
 
-1. In **Server**, enter the name for the data source.
+1. In **Server**, enter the name for the data source, such as your Oracle net service name (i.e., myADB_high) or Easy Connect Plus connection string.
 
-1. Under **Authentication method**, choose either **Windows** or **Basic**. Choose **Basic** if you plan to use an account you created within Oracle instead of Windows authentication. Then enter the credentials to use for this data source.
+1. Under **Authentication method**, choose either **Windows** or **Basic**. Choose **Basic** if you plan to login as an Oracle database user. Then enter the credentials to use for this data source. Choose **Windows** when using Windows operating system authentication and with both the Oracle client and server running on Windows.
```
Comment (Court72) [inline powerbi-docs/connect-data/service-gateway-onprem-manage-oracle.md:39]:
```suggestion
1. Under **Authentication method**, choose either **Windows** or **Basic**. Choose **Basic** if you plan to log in as an Oracle database user. Then enter the credentials to use for this data source. Choose **Windows** when using Windows operating system authentication and with both the Oracle client and server running on Windows.
```

File: powerbi-docs/connect-data/service-gateway-onprem-manage-oracle.md [hunk 2]
```diff
@@ -20,29 +20,23 @@ After you [install the on-premises data gateway](/data-integration/gateway/servi
...
+After you install and configure OCMT properly, you may use Power BI Desktop or another test client to verify correct installation and configuration on the gateway.
 
 ## Add a data source
 
 1. On the **New connection** screen for your on-premises data gateway, select **Oracle** for **Connection type**.
 
    :::image type="content" source="media/service-gateway-onprem-manage-oracle/add-data-source-oracle.png" alt-text="Screenshot of adding an Oracle data source to the gateway." :::
 
-1. In **Server**, enter the name for the data source.
+1. In **Server**, enter the name for the data source, such as your Oracle net service name (i.e., myADB_high) or Easy Connect Plus connection string.
```
Comment (Court72) [inline powerbi-docs/connect-data/service-gateway-onprem-manage-oracle.md:37]:
```suggestion
1. In **Server**, enter the name for the data source, such as your Oracle net service name (for example, myADB_high) or Easy Connect Plus connection string.
```

File: powerbi-docs/connect-data/service-gateway-onprem-manage-oracle.md [hunk 3]
```diff
@@ -20,29 +20,23 @@ After you [install the on-premises data gateway](/data-integration/gateway/servi
...
+- Oracle Autonomous Database - all versions
 
-When you install the client:
-
-- Choose an ODAC version that's compatible with your Oracle server. For instance, ODAC 12.x doesn't always support Oracle Server version 9.
-- Choose the Windows installer.
-- During setup, make sure to select the checkbox next to **Configure ODP.NET and/or Oracle Providers for ASP.NET at machine-wide level** in the setup wizard so that Power BI can connect to your Oracle database. Some versions of the setup wizard select this checkbox by default and others don't.
-
-After you install the client and configure ODAC properly, use Power BI Desktop or another test client to verify correct installation and configuration on the gateway.
+After you install and configure OCMT properly, you may use Power BI Desktop or another test client to verify correct installation and configuration on the gateway.
```
Comment (Court72) [inline powerbi-docs/connect-data/service-gateway-onprem-manage-oracle.md:29]:
```suggestion
After you install and configure OCMT properly, you can use Power BI Desktop or another test client to verify the correct installation and configuration on the gateway.
```

**Comment by maggiesMSFT** [comment]
> @maggiesMSFT I work at a partner. company, Oracle. I am not a MS employee, vendor, nor contractor.
> 
> Nonetheless, I tried to access the site with my MS credentials. It says I don't have permission to the resource.

Hi, @alexkeh -- got it. Thanks for explaining and for updating the docs!

**Pull request merged**